frontend/src/pages/projects/screens/detail/overview/trainModels/PipelinesCard.tsx

```diff
@@ -139,25 +139,18 @@ const PipelinesCard: React.FC = () => {
       return (
         <>
           <CardBody>
-            <Stack hasGutter>
-              <TextContent>
-                <Text component="small">
-                  Pipelines are machine-learning workflows that you can use to train your model. To
-                  create or import pipelines, you must first configure a pipeline server.
-                </Text>
-              </TextContent>
-              {notebooksLoaded && !notebooksError && notebooks.length > 0 ? (
-                <Alert
-                  isInline
-                  isPlain
-                  variant="warning"
-                  title="Restart running workbenches after configuring the pipeline server"
-                >
-                  If you’ve already created pipelines in a workbench, restart the workbench after
-                  configuring the pipeline server to view your pipelines here.
-                </Alert>
-              ) : null}
```

Comment:
@jeff-phillips-18 @jgiardino can we confirm whether the intent is to have the "If you've already created..." text inside an Alert? Latest rebase had a conflict with the updates recently made to this portion of the file.

Reply:
Yes. It should be in the alert.

Reply:
Here's the PR change you're looking for: frontend/src/pages/projects/screens/detail/overview/trainModels/PipelinesCard.tsx